tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -231,8 +231,10 @@ struct RNTupleLocator {
       kTypeFile = 0x00,
       kTypeDAOS = 0x02,
 
+      kTypeTestLocator = 0x7e, // used for unit tests
```

Comment:
for other test constants (e.g. `kTestFutureFieldStructure`), they were kept outside of the `enum` definition - do we need this here?